modules/signals/rxjs-interop/src/rx-method.ts

```diff
@@ -30,14 +31,21 @@ export function rxMethod<Input>(
   const sourceSub = generator(source$).subscribe();
   destroyRef.onDestroy(() => sourceSub.unsubscribe());
 
-  const rxMethodFn = (input: Input | Signal<Input> | Observable<Input>) => {
+  const rxMethodFn = (
+    input: Input | Signal<Input> | Observable<Input>,
+    config?: { injector?: Injector }
+  ) => {
     if (isSignal(input)) {
+      const callerInjector = getCallerInjectorIfAvailable();
+      const customInjector = config?.injector;
+      const instanceInjector = customInjector ?? callerInjector ?? injector;
```

Comment:
perf: can we skip the execution of `getCallerInjectorIfAvailable` when a `customInjector` is provided via the config?

```suggestion
      const instanceInjector = config?.injector ?? getCallerInjectorIfAvailable() ?? injector;
```

Reply:
Sure